src/clt_least_squares/CLTLeastSquares.jsx

```diff
@@ -110,6 +112,14 @@ export class CLTLeastSquares extends Component {
             return jStat.variance(residuals) * (99 / 98);
         });
     }
+    getPopulationSSE(population, populationRegression) {
```

Comment:
good candidate for a test? (I might just not be seeing where the test is btw)

Reply:
The real meat of this is in `calculateSSE`. I don't have a unit test for that, but I do call it in tests for the Least Squares sim.
In any case, this method, and `findLinearRegression` should have proper unit tests on them.